docs/advanced/vgpusupport.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_position: 9
+sidebar_label: vGPU Support
+title: "vGPU Support"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/advanced/vgpusupport"/>
+</head>
+
+_Available as of v1.3.0_
+
+Harvester now offers the capability to share NVIDIA GPU's supporting SRIOV based virtualisation as vGPU devices.
+
+The additional capability is provided by the `pcidevices-controller` addon, and leverages `sriov-manage` to manage the gpu. Please refer the [Nvidia Documentation](https://docs.nvidia.com/grid/15.0/grid-vgpu-user-guide/index.html#creating-sriov-vgpu-device-red-hat-el-kvm) and your GPU documentation to identify if the GPU is supported.
+
+
+The [nvidia-driver-toolkit](https://docs.harvesterhci.io/v1.3/advanced/nvidia-driver-toolkit) addon needs to be enabled for users to be able to manage the lifecycle of vGPU's on GPU devices.
+
+## Usage
+
+1. On the **Dashboard** screen of Harvester UI, navigate to `SR-IOV GPU Devices` and check if GPU devices have been scanned and an associated `sriovgpudevices.devices.harvesterhci.io` object has been created. Once device is found it can be enabled by the users
+
```

Comment:
Indention of picture is not correct here, please see the preview: https://65f124fc753a388ab7209308--harvester-preview.netlify.app/v1.3/advanced/vgpusupport#usage